### PR TITLE
New Magnet-UI

### PR DESCRIPTION
**Pull request by bidery**

### CODE REVIEW

File: src/UI/src/components/MessageList/MessageList.tsx [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+            ))}
+          </TableBody>
+        </Table>
+      </TableContainer>
+      <div className={classes.buttonPosition}>
+        <Button
+          variant="contained"
+          disabled={!hasNext}
+          onClick={() => {
+            loadNext(30);
```
[the remark this answers is not in the document]
Comment (bidery) [reply src/UI/src/components/MessageList/MessageList.tsx:272]: loadnext() needs a value. I cannot remove `30`

File: src/UI/src/components/MessageListTable/MessageListTable.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  const classes = useStyles();
+  const navigate = useNavigate();
+
+  const getDateTime = (date: any) => new Date(date).toLocaleString() ?? "";
+
+  const getShortTitle = (title: string) =>
+    title.length > 50 ? title.substring(0, 50) + "..." : title;
+
+  return (
+    <>
```
Comment (bejto) [inline src/UI/src/components/MessageListTable/MessageListTable.tsx:58]: remove

File: src/UI/src/components/ReceivedLogTable/ReceivedLogTable.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+const ReceivedLogTable: React.FC<ReceivedLog> = ({ receivedLog }) => {
+  const classes = useStyles();
+
+  const getDateTime = (date: any) => new Date(date).toLocaleString() ?? "";
+
+  return (
+    <>
+      <h1 className={classes.dataTitle}>Received Log</h1>
+      <TableContainer className={classes.tableMargin}>
+        <Table sx={{ minWidth: 650 }}>
```
Comment (bejto) [inline src/UI/src/components/ReceivedLogTable/ReceivedLogTable.tsx:39]: use UseStyles

File: src/UI/src/components/ReceiverList/ReceiverList.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+const useStyles = makeStyles({
+  dataTitle: {
+    fontSize: "20px",
+    marginTop: "35px",
+    fontWeight: "400",
+  },
+});
+
+interface Receivers {
```
Comment (bejto) [inline src/UI/src/components/ReceiverList/ReceiverList.tsx:20]: ReceiverListProps

File: src/UI/src/views/Home/Home.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    graphql`
+      query HomeQuery {
+        ...MessageListFragment_query
+      }
+    `,
+    { fetchPolicy: "store-or-network" }
+  );
+
+  return (
+    <>
```
Comment (bejto) [inline src/UI/src/views/Home/Home.tsx:19]: dont needed

File: src/UI/src/components/MessageListTable/MessageListTable.tsx (hunk 2 above)
Comment (bidery) [reply src/UI/src/components/MessageListTable/MessageListTable.tsx:58]: not possible, because there is a div under the table

**Review by bejto — APPROVED** [review]
well done :)

**Pull request merged**